recipes/ua-parser/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{%set name = "ua-parser" %}
+{%set version = "0.7.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "e1f7cb16ccb68d6c93b19348e678500bec8cd1f692e49214dd38ca2d100ada9b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
Sure, we have had this discussion before. There are some people that want to compile from the repo directly and some that don't. We ended up letting this slide with PyPI and such.